src/_components/EmailLink.jsx

```diff
@@ -0,0 +1,12 @@
+export default function EmailLink({ name, emailAddr, textColor }) {
+  return (
+    <p>
+      <strong className={`text-[${textColor}]`}>
```

Comment:
I'd recommend using the TailwindCSS utility [font-weight](https://tailwindcss.com/docs/font-weight) to handle bolding text instead of the <strong> tag, just to reduce nested tags in the file

Reply:
I removed this component altogether since it won't be used anymore.